cpp/tests/column/column_device_view_test.cu

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf/types.hpp>
+#include <cudf/column/column.hpp>
+#include <cudf/column/column_view.hpp>
+#include <cudf/column/column_device_view.cuh>
+#include <cudf/copying.hpp>
+
+#include <tests/utilities/base_fixture.hpp>
+#include <tests/utilities/column_wrapper.hpp>
+#include <tests/utilities/column_utilities.hpp>
+#include <tests/utilities/cudf_gtest.hpp>
+
+struct ColumnDeviceViewTest : public cudf::test::BaseFixture {};
+
+TEST_F(ColumnDeviceViewTest, Sample) {
+    using T = int32_t;
+    cudaStream_t stream = 0;
+    cudf::test::fixed_width_column_wrapper<T> input ({1, 2, 3, 4, 5, 6});
+    auto input_column(std::move(input.release()));
```

Comment:
No need to convert to column. `fixed_width_column_wrapper` can be passed as argument in place of `column_view` (implicit conversion happens).